ts/Series/Lollipop/LollipopSeries.ts

```diff
@@ -104,8 +120,30 @@ class LollipopSeries extends DumbbellSeries {
                     halo: false
                 }
             },
+            /** @ignore-option */
+            stickyTracking: false,
+            /** @ignore-option */
+            trackByArea: false,
+            /** @ignore-option */
+            fillColor: 'none',
+            /** @ignore-option */
+            lineWidth: 0,
+            crisp: false,
+            dataLabels: {
+                align: void 0,
+                verticalAlign: void 0,
+                /**
+                 * The y position offset of the label relative to the point in
+                 * pixels.
+                 *
+                 * @type {number}
+                 */
+                y: void 0
+            },
+            pointRange: 1,
             tooltip: {
-                pointFormat: '<span style="color:{series.color}">●</span> {series.name}: <b>{point.y}</b><br/>'
+                headerFormat: '<span style="font-size: 10px">{point.key}</span><br/>',
+                pointFormat: '<span style="color:{series.color}">\u25CF</span> {series.name}: <b>{point.y}</b>'
```

Comment:
Seems like we're inheriting scatter series options only to resett to default line/column options... Can we extend the `Series` or `ColumnSeries.defaultOptions` instead?

Reply:
... Or maybe it makes more sense that the whole dumbbell inherits `Series` directly? In some ways it is more closely related, it only removes the `drawGraph` and adds the connectors?

Reply:
We could do it in this PR too, since it's closely related?

Reply:
@hubertkozik I made a typo. I meant "maybe it makes more sense that the whole _lollipop_ inherits Series directly?". Sorry for that. But for the lollipop, it seems to me that this is just a common Series with no graph, and connectors connecting the marker with the threshold. And then there's some ColumnSeries logic to handle how multiple points are laid out on the same x value.

Reply:
Ok, got it, thank you for your clarification. I feel, that it should inherit from `ScatterSeries` because it looks like a point with a line back to the threshold. Also, I think, that it makes sense to inherit default options from `Series.defaultOptions` and fill them with other necessary options for Lollipop. What do you think about it? I have pushed new changes (with inheriting from `ScatterSeries`).